api/shape/main.js

```diff
@@ -355,6 +355,7 @@ export { z } from './z.js';
 export { Alpha } from './Alpha.js';
 export { Arc } from './Arc.js';
 export { Assembly } from './Assembly.js';
+export { Assembly2 } from './Assembly2.js';
```

Comment:
So it's actually called Assembly2 in the api? Like to test I should call it that way?

Reply:
Yeah, I wanted to keep it an experimental interface for now, since it doesn't address the general problem -- just volumes for now.